### PR TITLE
Easter is Out!

### PR DESCRIPTION
**Pull request by sawu-tg**
- Chickens have been refactored to be entirely interchangeable
- Rabbits have been added, they will bounce around and place eggs, even in space!
- A Rabbit Suit has been added, and can be won from opening easter eggs!
- Easter Baskets, allowing for easy carry of Easter Eggs!
- Easter Eggs will spawn in maintenance
- Easter Baskets also spawn in maintenance.
- Delicious Easter Recipes (Mammi, Hot Cross Buns, Brioche, Scotch Eggs and Chocolate Bunnies)

rabbitstuff

(Suit, New Rabbits and Egg Basket by @Nienhaus)

### CODE REVIEW

**Comment by RemieRichards** [comment]
The Travis issue isn't a code one.
it passed 2/3 Builds.

**Comment by vista-** [comment]
>no chance for xeno bunnies which actually lay alien eggs

3/10

**Comment by Anonus** [comment]
Space rabbits need to ignore space drifting like carp do or you'll have bunnies drifting into space.

**Comment by sawu-tg** [comment]
@Anonus Bunnies drifting was intentional, Space Carp are made for space, Bunnies are not. They spawn with a cute little helmet, but have no jetback or anything, so they'll just drift around in space, pooping out eggs, which also drift.

That and Rabbit-Meteors smashing into the station.

File: code/modules/holiday/easter.dm [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+/datum/round_event/rabbitrelease/start()
+	for(var/obj/effect/landmark/R in landmarks_list)
+		if(R.name != "blobspawn")
+			if(prob(35))
+				if(istype(R.loc,/turf/space))
+					new /mob/living/simple_animal/chicken/rabbit/space(R.loc)
+				else
+					new /mob/living/simple_animal/chicken/rabbit(R.loc)
+
+/mob/living/simple_animal/chicken/rabbit
```
Comment (Cheridan) [inline code/modules/holiday/easter.dm]: lol

Comment (sawu-tg) [inline code/modules/holiday/easter.dm]: I call them Rickens.. Or Cabbits. I'unno blame Nanotrasen and their crackpot genetics team.

**Pull request merged**